processing/src/main/java/io/druid/segment/QueryableIndexStorageAdapter.java

```diff
@@ -213,16 +229,59 @@ public DateTime getMaxIngestedEventTime()
       actualInterval = actualInterval.withEnd(dataInterval.getEnd());
     }
 
-    final Offset offset;
+    final ColumnSelectorBitmapIndexSelector selector = new ColumnSelectorBitmapIndexSelector(
+        index.getBitmapFactoryForDimensions(),
+        index
+    );
+
+    Offset offset;
+    List<Filter> postFilters = new ArrayList<>();
```

Comment:
A short comment about what pre/post filters mean would help people looking at this in the future.